src/routes/users.route.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import UsersController from '../controllers/users.controller';
+
+const router = express.Router();
+const usersController = new UsersController();
+
+router.use(express.json());
```

Comment:
This line is usually put in server index file, `src/app.js` in your case.
`router.use(express.json());` applies middleware for all routes, not only specific one.

Reply:
The idea was to create separate router instance for each route with its own set of middleware as an independent unit of application. router.use(express.json()); works only for user.router, and I need to add this line for any new router if this router is going to work with req.body. I don't wont to have json parser on the app layer.

Reply:
but if you react API service, you're going to use json on all routes, don't you?